tasks/start.js

```diff
@@ -3,13 +3,40 @@
 var childProcess = require('child_process');
 var electron = require('electron-prebuilt');
 var gulp = require('gulp');
+var extend = require('util')._extend;
+var utils = require('./utils');
+
+var buildPath = './build';
 
 gulp.task('start', ['build', 'watch'], function () {
-    childProcess.spawn(electron, ['./build'], {
-        stdio: 'inherit'
-    })
-    .on('close', function () {
-        // User closed the app. Kill the host process.
-        process.exit();
+
+    utils.initBrowserSync({port: 3002}, function(bs) {
```

Comment:
Yes, please. It makes more sense from separation of concernes standpoint. You can also create totally separate file for browserSync and just require it in start. Might be even better this way.